captum/attr/_core/deep_lift.py

```diff
@@ -102,16 +104,32 @@ def attribute(
                         to the number of examples (aka batch size), and if
                         multiple input tensors are provided, the examples must
                         be aligned appropriately.
-            baselines (tensor or tuple of tensors, optional): Baselines define
-                        reference samples which are compared with the inputs.
-                        In order to assign attribution scores DeepLift computes
-                        the differences between the inputs and references and
-                        corresponding outputs.
-                        If inputs is a single tensor, baselines must also be a
-                        single tensor with exactly the same dimensions as inputs.
-                        If inputs is a tuple of tensors, baselines must also be
-                        a tuple of tensors, with matching dimensions to inputs.
-                        Default: zero tensor for each input tensor
+            baselines (scalar, tensor, tuple of scalars or tensors, optional):
+                        Baselines define reference samples that are compared with
+                        the inputs. In order to assign attribution scores DeepLift
+                        computes the differences between the inputs and references
+                        and corresponding outputs.
+                        Baselines can be provided either as :
+
+                        - a single tensor, if inputs is a single tensor, with
+                            exactly the same dimensions as inputs or the first
+                            dimension is one and the remaining dimensions match
+                            with inputs.
+
+                        - a tuple of tensors, if inputs is a tuple of tensors,
```

Comment:
nit: This probably should be updated as well to say that the first dimension of any of them in the tuple could be 1? Also, from the implementation, it looks like you can also mix tensors (with first dimension 1 or full dimension) and scalars, can also document that if it's true?